lib/util.js

```diff
@@ -92,10 +99,23 @@ function escapeStyleCode(code) {
 /**
  * @param {string | string[]} format
  * @param {string} text
+ * @param {object} [options={}]
+ * @param {boolean} [options.validateStream=true] - Whether to validate the stream.
+ * @param {Stream} [options.stream=process.stdout] - The output stream to write the styled text to.
  * @returns {string}
  */
-function styleText(format, text) {
+function styleText(format, text, { validateStream = true, stream = process.stdout } = {}) {
   validateString(text, 'text');
+
+  if (validateStream) {
+    if (
+      !stream ||
```

Comment:
I think we should throw an error if `validateStream` is true but `stream` is not a stream.

Reply:
Or maybe better, always validate the type of `stream`.

Reply:
Done in [cc08b60](https://github.com/nodejs/node/pull/54389/commits/cc08b60053766f5224d2aa95fdc27135e1933412)

I'm not sure if we want to validate `isReadable`, but I've added anyway.